ci/properties/dlang.properties.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "D Programming Language",
+    "description": "Build, run and test a D Programming Language project with Dub.",
+    "iconName": "dlang",
+    "categories": ["D Programming Language"]

```

```suggestion
    "categories": [ "D" ]
```

I looked up other languages and none is using "Programming Language" or "lang" suffix